config.ini.example

```diff
@@ -9,4 +9,4 @@ username = user
 update_cal_location = /home/user/upstate_tech_cal_service/update_cal_data.py
 
 [past_events]
-past_days = 5
\ No newline at end of file
+days_in_the_past = 5
```

Comment:
Has this name been changed in app.py too where it's getting the value from the config.ini?

Reply:
oops! I didn't have that in these commits. Let me add them now.